src/@types/Options.ts

```diff
@@ -1,5 +1,7 @@
 import type { Context } from './Context'
-import { Generator } from './Generator'
+import type { Generator } from './Generator'
+import type { GetServer } from './GetServer'
+import type { LifeCycleType } from 'elysia/dist/types'
```

Comment:
```suggestion
```

as i said, even Elysia docs nowhere mentioned about `LifeCycleType` this would lead to more confusion